code/API_definitions/Discovery/README.md

```diff
@@ -1,50 +1,60 @@
+
+This API allows a client application to discover the closest MEC platform to the UE hosting the client application. 'Closest' means 'shortest network path' as that will give the shortest propagation distance, which is a major factor in latency.
 
 ## MEC Experience Management and Exposure API
+
 This API allows a developer to:
+
 - discover available MEC platforms, ranked by proximity to a UE.
 - read the state (availability and capabilities) of an operator's various MEC platforms.
-- register a service profile (a description of the developer's edge service) with the MEC operator
-- register the deployed service endpoints with the MEC operator, which allows the closest service endpoint to be discovered at runtime
+- register a service profile (a description of the developer's edge service) with the MEC operator.
+- register the deployed service endpoints with the MEC operator, which allows the closest service endpoint to be discovered at runtime.
 
-The API will also support the following capabilities: 
-- events(such as change of status of a MEC platform or another event which could affect their service)
+The API will also support the following capabilities:
+
+- events(such as change of status of a MEC platform or another event which could affect their service).
 - subscription to notification of events.
 
-# Mapping to the list of intents
+## Mapping to the list of intents
 
 These APIs fulfil the ['discovery' intents](https://github.com/camaraproject/EdgeCloud/blob/main/documentation/SupportingDocuments/Harmonisation%20of%20APIs/describing%20and%20harmonising%20the%20Edge%20APIs.md)
 
-*Simple Edge Discovery* fulfils a single intent,  "4. I can discover the closest MEC platform to a specific terminal (closest in terms of shortest network path)"
+*Simple Edge Discovery* fulfils a single intent, "4. I can discover the closest MEC platform to a specific terminal (closest in terms of shortest network path)"
 
 *MEC Exposure and Experience Management* is a more comprehensive discovery API and fulfils the following intents:
 
 ### Developer intents
-#### Provisioning intents 
-1.	“I can retrieve a list of the operator’s MECs and their status, ordering the results by location and filtering by status (active/inactive/unknown)”
-2.	"I can discover the capabilities/resources available at an operator’s MEC: CPU, Memory, Storage, GPU"
-3.	"I can discover the geographical regions covered by the operators MECs"
-4.	"I can discover the closest MEC platform to a specific terminal (closest in terms of shortest network path)"
 
-16.	“I can ask the operator to provide the details of all the onboarded applications”
-17.	"I can ask the operator to inform about the application instance details e.g., communication endpoints, resource consumed etc"
+#### Provisioning intents
+
+1. "I can retrieve a list of the operator’s MECs and their status, ordering the results by location and filtering by status (active/inactive/unknown)"
```

Comment:
We have terms with alternate words e.g. here it says MECs but the API says MEC Platforms